web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -15,6 +19,21 @@ export default function EstimationCard({
   estimatedPercentage?: number;
   outageMessage: ZoneDetails['zoneMessage'];
 }) {
+  const [currentlyShowingFeedbackCard, setcurrentlyShowingFeedbackCard] = useState(false);
+  const [collapsedNumber, setCollapsedNumber] = useState(0);
+  const feedbackEnabled = useFeatureFlag('feedback-estimation-labels');
+
+  useEffect(() => {
+    setcurrentlyShowingFeedbackCard(
+      feedbackEnabled &&
+        showEstimationFeedbackCard(
+          collapsedNumber,
+          currentlyShowingFeedbackCard,
+          localStorage.getItem('feedbackCardStatus')
+        )
+    );
+  }, [feedbackEnabled, collapsedNumber]);
```

Comment:
I added the feature-flag wrapping here, it was super easy thanks to the way you made it 🙌 

Here is an example :
https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/5f26d852-d075-4b25-a489-bccd278836a1

Reply:
Cool! I didn't know that &&ff after remote=true would show that. What makes the &&ff work like that?

Reply:
We have this component that only shows when the query parameter is used :)

![Screenshot 2024-02-27 at 10 23 05](https://github.com/electricitymaps/electricitymaps-contrib/assets/3296643/f42c7ad9-e866-43bf-ada2-1a092d4f7892)

Reply:
@madsnedergaard have you actually documented this anywhere? Might be good to do for our future selfs.

_I didn't know about this particular feature either..._